### PR TITLE
Local environment unit tests

### PR DESCRIPTION
**Pull request by xBalbinus**
## PR Checklist

- [x] Added Tests
- [ ] Updated Documentation
- [ ] Updated CHANGELOG.md for the appropriate package

### CODE REVIEW

File: packages/local-environment/tests/nomadenv.test.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        expect(le.bridgeSDK).to.exist;
+        expect(le.coreSDK).to.exist;
+
+        // Can add domains
+        le.addDomain(tDomain);
+        le.addDomain(jDomain);
+        assert.isTrue(le.domains.includes(tDomain));
+        assert.isTrue(le.domains.includes(jDomain));
+
+        // Can up agents
```
Comment (yourbuddyconner) [inline packages/local-environment/tests/nomadenv.test.ts:30]: Here I might go so far as to check if docker containers are running as opposed to checking `isAgentUp` alone

Comment (xBalbinus) [reply packages/local-environment/tests/nomadenv.test.ts:32]: Implemented!

Comment (xBalbinus) [reply packages/local-environment/tests/nomadenv.test.ts:32]: @yourbuddyconner Thoughts on running tests in order? Tests are now running one by one but I'm not exactly sure as to how to specify the order in which the tests run.

File: .github/workflows/tests-integration.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+name: Build and Test Packages
+on: [push]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Foundry
+        uses: onbjerg/foundry-toolchain@v1
```
Comment (rswanson) [inline .github/workflows/tests-integration.yml:9]: Could you change it from onbjerg/foundry-toolchain@v1 to `foundry-rs/foundry-toolchain@v1` - I know onbjerg created foundry so it mostly fine but safer to use the official version

Comment (xBalbinus) [reply .github/workflows/tests-integration.yml:9]: On it

**Comment by xBalbinus** [comment]
All tests passing.  --detectOpenHandles should be able to exit jest tests after running

File: packages/local-environment/tests/nomadenv.test.ts (hunk 1 above)
Comment (yourbuddyconner) [reply packages/local-environment/tests/nomadenv.test.ts:37]: I don't think order matters, especially if tests clean up after themselves

**Review by rswanson — APPROVED** [review]
lgtm

**Review by kekonen — APPROVED** [review]
LGTM STAAAAAAAMPPP!

**Review by yourbuddyconner — APPROVED** [review]
Test output LGTM -- Note to Self: need to schedule a project to clean up the output of the Deploy script

**Comment by odyslam** [comment]
@yourbuddyconner What is the issue with the deploy script? We just spent a bunch of time on the output with @anna-carroll  and I was actually very happy with the result 🥺 . Huge UX improvement over the last iteration, especially in error handling and printing.

**Comment by yourbuddyconner** [comment]
TLDR console.logs need to be replaced with bunyan logging in the deploy output at minimum
